modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm

```diff
@@ -137,15 +140,23 @@
 		to_chat(owner, span_warning("You already know this chemical!"))
 		cortical_owner.chemical_evolution += 5
 		return
+	if(locate(reagent_choice) in cortical_owner.blacklisted_chemicals)
+		to_chat(owner, span_warning("Your physiology is incompatible with this chemical - find it elsewhere!"))
```

Comment:
remove the "find it elsewhere" since they can't learn it whatsoever-- which I'm still against blacklisting chems (beyond non-synthesizable ones).